core/src/main/java/com/graphhopper/routing/util/AbstractFlagEncoder.java

```diff
@@ -426,6 +427,11 @@ public DecimalEncodedValue getDecimalEncodedValue(String key) {
     public <T extends Enum> EnumEncodedValue<T> getEnumEncodedValue(String key, Class<T> enumType) {
         return encodedValueLookup.getEnumEncodedValue(key, enumType);
     }
+    
+    @Override
+    public SpatialRuleLookup getSpatialRuleLookup() {
+        return encodedValueLookup.getSpatialRuleLookup();
+    }
```

Comment:
Depends on the fact if we switch `PathDetailsBuilderFactory.createPathDetailsBuilders()` to accept an EncodingManager instead of the EncodedValueLookup interface.